docs/intro-to-dashboards-and-apps/connecting-etl-to-dashboard-with-voila.md

```diff
@@ -6,9 +6,178 @@ jupyter:
+
+Specifically, `SelectMultiple`, `Dropdown`, and `Combobox` widgets are employed to filter categorical columns, including the car's fuel type, size, model, and model year. Note that `Combobox`, which is a [String widget](https://ipywidgets.readthedocs.io/en/stable/examples/Widget%20List.html#combobox). was not introduced earlier in the course and we recommend taking a look at its documentation. This course The $CO_2$ ratings column (a higher rating suggesting lower $CO_2$ emissions) can also be filtered with the `IntSlider` widget.
+
+```{important}
+During this process, it is essential to become acquainted with the various types of data present in each table, such as numerical, categorical, boolean, and others, before proceeding with data visualization. This familiarity aids the user in developing an intuition about the most suitable visualizations for emphasizing columns individually or exploring relationships between them.
+```
+
+### Questions Answered through Visualizations
+
+After the user interacts with the tables, questions that the dashboard answers with the help of visualizations are then listed (see above). It is recommended to begin with simpler, more high-level questions about the data, such as addressing counts of different vehicle types or distributions of emissions, before diving into relationships between columns. This allows the user to make more informed decisions about the types of visualizations to use for answering more complex questions.
+
+### Interactive Visualizations
+
+#### `Seaborn` Bar Plots
+
+The first visualization is a `barplot()`, built with `seaborn` and `ipywidgets`, of the number of unique fuel-only, electric, and hybrid car models, visualized across the years they were released. A trend overlay line (`pointplot()`) is added to clearly spot yearly manufacturing patterns in these vehicles. Interactivity is enabled through `RadioButtons`, which helps the user toggle between a bar plot of either fuel-only cars or a column chart of hybrid and electric cars. The decision to provide this separation is because fuel-only cars' models date back to 1995 in the dataset, while hybrid and electric cars' models date back to 2012. Therefore, the previous recommendation that the user should get familiar with the data before visualizing it is important here, which also assists in thinking about both the type of widget and where it could be utilized with respect to the data.
```

Comment:
Embed visualization into blog, otherwise it is a lot of text for the user to parse

Reply:
fixed